### DISCUSSION TITLE
What would cuase the check to fail?

### DISCUSSION THREAD

**Discussion by miguelarios** [cupcakearmy/autorestic#277] [Q&A]
I am seeing that sometimes my check command will fail. I don't know why. What can you do to fix? I am just having to delete my repo and start again the check to init it.

It seems that the backup will still work tho...

```
root@EDITH:~# autorestic -c /mnt/user/appdata/autorestic/.autorestic.yml check --ci
Using config: 	 /mnt/user/appdata/autorestic/.autorestic.yml
Using lock:	 /mnt/user/appdata/autorestic/.autorestic.lock.yml
> Executing: /usr/local/bin/restic check
Initializing backend "remote"...
> Executing: /usr/local/bin/restic init
Error: exit status 1
```

**Comment by ssanj** [comment]
I've seen this happen if I don't have enough privileges to access the repo. Do you create the repository using `sudo` ?

**Reply by perry-mitchell** [reply]
I'm not sure that sudo would help being that it's initialising a _remote_ backend by the looks of it. I'm having the same issue with an S3 compatible backend.. init just fails with `1`.

**Reply by zageyiff** [reply]
run with verbose flag, might give you more hints
```bash
$ autorestic check --verbose
Using config paths: . /home/don /home/don/.config/autorestic
Using config: 	 /home/don/.autorestic.yml
Using lock:	 /home/don/.autorestic.lock.yml
> Executing: /usr/bin/restic check
Initializing backend "local"...
> Executing: /usr/bin/restic init
```

In my case it's failing as well, I'm struggling to pass the password via env `AUTORESTIC_LOCAL_RESTIC_PASSWORD` for a backend named local
Following the code, the `/usr/bin/restic check` command somehow fails, and then autorestic tries to initialize and already initialized repo.
